OpenEdXMobile/src/main/java/org/edx/mobile/viewModel/InAppPurchasesViewModel.kt

```diff
@@ -84,31 +86,33 @@ class InAppPurchasesViewModel @Inject constructor(
 
     init {
         billingProcessor.setUpBillingFlowListeners(listener)
-        billingProcessor.startConnection()
     }
 
     fun initializeProductPrice(courseSku: String?) {
```

Comment:
can we make the courseSku nonNull as without SKU the button won't be visible.

Reply:
we can check it in Oberser before passing it to any method.
it has only 2 or 3 implementations